### PR TITLE
[TACHYON-1207] Added test coverage for RecomputeExecutor

### PR DESCRIPTION
**Pull request by yupeng9**
https://tachyon.atlassian.net/browse/TACHYON-1207

### CODE REVIEW

File: servers/src/test/java/tachyon/master/lineage/recompute/TestRecomputeExecutor.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+/**
+ * Tests {@link RecomputeExecutor}.
+ */
+@RunWith(PowerMockRunner.class)
+@PrepareForTest(FileSystemMaster.class)
+public final class TestRecomputeExecutor {
+  @Test
+  public void recomputeLauncherTest() throws Exception {
+    long fileId = 5L;
+    // mock planner
```
Comment (aaudiber) [inline servers/src/test/java/tachyon/master/lineage/recompute/TestRecomputeExecutor.java]: Do you think the `// mock x` comments are worth?

File: servers/src/test/java/tachyon/master/lineage/recompute/TestRecomputeExecutor.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+import tachyon.master.lineage.meta.LineageFileState;
+
+/**
+ * Tests {@link RecomputeExecutor}.
+ */
+@RunWith(PowerMockRunner.class)
+@PrepareForTest(FileSystemMaster.class)
+public final class TestRecomputeExecutor {
+  @Test
+  public void recomputeLauncherTest() throws Exception {
```
Comment (aaudiber) [inline servers/src/test/java/tachyon/master/lineage/recompute/TestRecomputeExecutor.java]: Can you add a quick javadoc for what the test is testing?

File: servers/src/test/java/tachyon/master/lineage/recompute/TestRecomputeExecutor.java [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+import tachyon.client.file.TachyonFile;
+import tachyon.job.Job;
+import tachyon.master.file.FileSystemMaster;
+import tachyon.master.lineage.meta.Lineage;
+import tachyon.master.lineage.meta.LineageFile;
+import tachyon.master.lineage.meta.LineageFileState;
+
+/**
+ * Tests {@link RecomputeExecutor}.
```
Comment (aaudiber) [inline servers/src/test/java/tachyon/master/lineage/recompute/TestRecomputeExecutor.java]: +1 for linking to the tested class.

**Comment by aaudiber** [comment]
@yupeng9 Thanks for improving test coverage! I left a few minor comments.

**Comment by gpang** [comment]
LGTM

File: servers/src/test/java/tachyon/master/lineage/recompute/TestRecomputeExecutor.java (hunk 2 above)
Comment (yupeng9) [inline servers/src/test/java/tachyon/master/lineage/recompute/TestRecomputeExecutor.java]: done

File: servers/src/test/java/tachyon/master/lineage/recompute/TestRecomputeExecutor.java (hunk 1 above)
Comment (yupeng9) [inline servers/src/test/java/tachyon/master/lineage/recompute/TestRecomputeExecutor.java]: I think it helps indicate the sections to improve the readability. What do you think?

Comment (aaudiber) [inline servers/src/test/java/tachyon/master/lineage/recompute/TestRecomputeExecutor.java]: I usually see it as a flag when the comment doesn't tell me anything more than reading the line of code it's commenting, sort of along the lines of `int i = 0; // initialize i to zero`, but I don't feel strongly about it.

Comment (yupeng9) [inline servers/src/test/java/tachyon/master/lineage/recompute/TestRecomputeExecutor.java]: I see. In that case, I think a blank line may also work.

**Comment by aaudiber** [comment]
LGTM

**Pull request merged**